VideoLocker/res/layout/view_register_spinner.xml

```diff
@@ -3,32 +3,20 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:orientation="vertical"
-    android:paddingTop="4dp"
-    android:paddingBottom="2dp">
+    android:orientation="vertical">
 
     <org.edx.mobile.module.registration.view.RegistrationOptionSpinner
         android:id="@+id/input_spinner"
-        style="@style/spinner"
+        style="@style/Base.Widget.AppCompat.Spinner.Underlined"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:textColorHint="@color/hint_grey_text"
-        android:paddingLeft="5dp"
-        android:paddingStart="5dp"
-        android:paddingRight="10dp"
-        android:paddingEnd="10dp"
+        android:textColorHint="@color/grey_text"
         android:focusable="true"
+        android:paddingLeft="-10dp"
         tools:targetApi="17"/>
 
     <TextView
         android:id="@+id/input_spinner_error"
         style="@style/registration_error_message"
-        android:visibility="visible"
-        tools:text="error text here"/>
```

Comment:
Having this turns out to be beneficial while viewing the layout in Android Studio's layout viewer, so, it shouldn't be removed.